src/Symfony/Component/HttpFoundation/RedirectResponse.php

```diff
@@ -76,6 +83,10 @@ public function getTargetUrl()
      */
     public function setTargetUrl($url)
     {
+        if (!is_string($url)) {
```

Comment:
If we were to be on PHP7, the cast to string would be done by the engine. What about just doing: `$url = (string) $url` ?